addons/mass_mailing/migrations/13.0.2.0/post-migration.py

```diff
@@ -0,0 +1,95 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_unlink_by_xmlid = [
```

Comment:
Not need to remove manual records which have noupdate=0 (views, actions, access rules, munes etc)

Reply:
access rules i think might be a problem when granted for any user. isn't it true?

Reply:
If they are noupdate=0, they are auto-removed, so no problem on that side.